static/js/actions/index.js

```diff
@@ -35,10 +36,10 @@ export const receiveDashboardFailure = (errorInfo: APIErrorInfo): Action => ({
 export const CLEAR_DASHBOARD = 'CLEAR_DASHBOARD';
 export const clearDashboard = () => ({ type: CLEAR_DASHBOARD });
 
-export function fetchDashboard(noSpinner: boolean = false): Dispatcher<Dashboard> {
+export function fetchDashboard(username: string, noSpinner: boolean = false): Dispatcher<Dashboard> {
   return (dispatch: Dispatch) => {
     dispatch(requestDashboard(noSpinner));
-    return api.getDashboard().
+    return api.getDashboard(username).
       then(dashboard => dispatch(receiveDashboardSuccess(dashboard))).
```

Comment:
Provided that makes sense to you. All I've changed is on the backend we can now theoretically get the information for another user, but on the frontend all the calls to `getDashboard` use the user's own username. The next PR I do will change the reducer to support multiple dashboards (similar to the setup we have for profiles) and then after that one I will probably work on actually adding the dashboard support to the learner page (along with the UI work, some of which I've already started).